Makefile

```diff
@@ -0,0 +1,59 @@
+.PHONY: dockerbuild dockerpush test testonce ruff black lint isort pre-commit-check requirements-update requirements setup
+VERSION ?= latest
+IMAGENAME = eodhp-web-presence
+DOCKERREPO ?= public.ecr.aws/n1b3o1k2/ukeodhp
+
+dockerbuild:
+	DOCKER_BUILDKIT=1 docker build -t ${IMAGENAME}:${VERSION} .
+
+dockerpush: dockerbuild testdocker
+	docker tag ${IMAGENAME}:${VERSION} ${DOCKERREPO}/${IMAGENAME}:${VERSION}
+	docker push ${DOCKERREPO}/${IMAGENAME}:${VERSION}
+
+test:
+	./venv/bin/ptw configscanning
+
+testonce:
+	./venv/bin/pytest
+
+ruff:
+	./venv/bin/ruff check .
+
+black:
+	./venv/bin/black .
+
+isort:
+	./venv/bin/isort . --check --diff --profile black
```

Comment:
isort also needs to be in the dev dependencies for this to work.

Reply:
Added